Specs/Scene/MetadataClassPropertySpec.js

```diff
@@ -103,7 +103,7 @@ describe("Scene/MetadataClassProperty", function () {
       property: {
         name: "Population",
         description: "Population (thousands)",
-        type: "INT32",
+        componentType: "INT32",
```

Comment:
This test is for `EXT_feature_metadata`, it should be `type`. The other change in this file is fine.